integration/src/test/java/org/mule/test/integration/exceptions/OnErrorContinueTestCase.java

```diff
@@ -186,6 +185,28 @@ public void onErrorTypeMatchSeveral() throws Exception {
     assertThat(getPayloadAsString(result), is("false apt1 apt2"));
   }
 
+  @Test
+  public void onErrorTypeMatchNameWildcard() throws Exception {
+    Message result = flowRunner("onErrorTypeMatchNameWildcard").withPayload(true).run().getMessage();
+    assertThat(result, is(notNullValue()));
+    assertThat(getPayloadAsString(result), is("true apt1 apt2"));
```

Comment:
There is a`org.mule.functional.junit4.matchers.MessageMatchers#hasPayload` than can be useful.

Reply:
hasPayload used